modules/ossm-release-notes-3-2-known-issues.adoc

```diff
@@ -0,0 +1,17 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/ossm-release-notes.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-release-3-2-known-issues_{context}"]
+= {SMProductName} 3.2 known issues
+
+[id="ambient-mode-not-supported-on-fips-enabled-openshift-clusters_{context}"]
+== Ambient mode not supported on FIPS-enabled OpenShift clusters
+
+{istio} ambient mode does not currently support {ocp-short-name} clusters running in Federal Information Processing Standards (FIPS) mode. Deployments that require FIPS compliance must continue using sidecar mode until support becomes available in a future release.
+
+[id="limitation-due-to-ztunnel-concurrency-issue_{context}"]
+== Limitation due to `ztunnel` concurrency issue
+
+A concurrency issue in `ztunnel` limits throughput scalability in {istio} ambient mode. Performance remains comparable to sidecar mode in most scenarios, but the issue may limit the potential to scale throughput performance.
```

Comment:
@longmuir do you think we can include the link to the upstream issue here?
https://github.com/istio/ztunnel/issues/1174